app/views/finance/npq/payment_breakdowns/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<% content_for :title, t("finance.npq.payment_breakdown") %>
+
+<h1 class="govuk-caption-xl">Coming soon!</h1>
```

Comment:
Probably needs a pass through Rob

Reply:
Rob says "ask Andy". Andy says "It's ok".